hack/generate_template.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
Do we really want to be duplicating this from `managed-cluster-config`? Seems like we would want to maintain this script (and even its associated framework) in one place.

Reply:
These `hack/generate_template.py` scripts are located in every repo that is set up to be deployed through the app-sre pipeline. See https://github.com/openshift/cloud-ingress-operator/blob/master/hack/generate_template.py as an example. 

These scripts take the resources that are in each repo's `deploy/` folder and bundles them into one template (usually `hack/olm-registry/olm-artifacts-template.yaml`) to be applied to Hive environments, which usually contains a SelectorSyncSet object that ensures that the objects in the `deploy/` folder are all deployed on each OSD cluster.

This repo deviates slightly from that because I do not want the CronJob deployed to OSD clusters - I want it only deployed to the Hive environments. 

The idea of maintaining this script (and its associated framework) in one place, as opposed to duplicating it across each repo that uses the app-sre pipeline for deployment, would be a *major* change to our infrastructure and pipeline, and is *well* outside the scope of this repo/PR/conversation.

Reply:
Thanks for the response.

I had only seen this thing in m-c-c, wasn't aware it was in lots of places (though I guess I'm not surprised).

We're clearly seeing a case-in-point for why central management of this framework would be beneficial (python2 is dead). But yeah, I get that that's totally out of scope here.